x-pack/plugins/index_lifecycle_management/common/types/policies.ts

```diff
@@ -80,6 +80,7 @@ export interface SerializedHotPhase extends SerializedPhase {
      * Only available on enterprise license
      */
     searchable_snapshot?: SearchableSnapshotAction;
+    rollup?: RollupAction;
```

Comment:
Would you mind moving this line up, so that it doesn't read like both searchable snapshots and rollup require enterprise license?